packages/experiments/src/components/Nav/SlimNav.tsx

```diff
@@ -236,11 +249,22 @@ class SlimNavComponent extends NavBase {
       return null;
     }
 
+    const {
+      enableCustomization,
+      showMore
+    } = this.props;
+
     return (
       <ul role='list'>
         {
           links.map((link: INavLink, linkIndex: number) => {
-            return this._renderLink(link, linkIndex, nestingLevel);
+            if (enableCustomization && link.isHidden && !showMore) {
```

Comment:
this is nit but it seems `Nav` and `SlimNav` could share this logic.

Reply:
will move to NavBase